.codecov.yml

```diff
@@ -1,11 +1,13 @@
 codecov:
   notify:
     require_ci_to_pass: yes
+  ignore:
+  - "tasks/*.py"
 
 coverage:
-  precision: 0
-  round: nearest
-  range: "70...95"
+  precision: 2
+  round: down
+  range: "70...100"
```

Comment:
Technically everything above 90% is fine. Hence why the badge is mostly green now too. I think aspiring to 100 is a good target, even if we'll not reach it. I've put the precision to 2 so that people covering new lines can straight away see the improvement, even if it's small.